src/Resources/views/show.xml.twig

```diff
@@ -15,5 +15,19 @@
             {{ xml_helper.change_frequency(url) }}
             {{ xml_helper.priority(url) }}
         </url>
+        {% if hreflang is not same as(false) %}
+            {% for locale, location in url.alternatives %}
+                <url>
+                    <loc>{{ url_helper.absolute_or_relative(location, absolute_url) }}</loc>
+                    <xhtml:link rel="alternate" hreflang="{{ language_helper.localeToCode(sylius.localeCode) }}" href="{{ url_helper.absolute_or_relative(url.localization, absolute_url) }}"/>
+                    {% for localeSub, locationSub in url.alternatives if localeSub is not same as(locale) %}
```

Comment:
According to Google "the Spanish version must include a rel="alternate" hreflang="x" link for itself in addition to links to the French and English versions. Similarly, the English and French versions must each include the same references to the French, English, and Spanish versions."

According to this we should remove the if statement on this row

Reply:
So that means that some lines up (in the first `<url>`) a reference to itself also needs to be included I think.